tests/filesTest.js

```diff
@@ -151,24 +150,21 @@ describe('Main: Currently testing files management,', function () {
                   'd:getlastmodified', '',
                   MatchersV3.date('EEE, d MMM yyyy HH:mm:ss z', 'Mon, 19 Oct 2020 03:50:00 GMT')
                 )
-                .appendElement('d:getcontentlength', '', MatchersV3.number(11))
                 .appendElement('d:resourcetype', '', resourceType)
                 .appendElement(
                   'd:getetag', '',
-                  MatchersV3.regex('"[a-z0-9]+"', '"3986cd55c130a4d50ff0904bf64aa27d"')
+                  MatchersV3.string('&quot;3986cd55c130a4d50ff0904bf64aa27d&quot;')
                 )
-                .appendElement('d:getcontenttype', '', 'text/plain')
+              if (item.type === 'file') {
+                dProp
+                  .appendElement('d:getcontentlength', '', MatchersV3.number(11))
+                  .appendElement('d:getcontenttype', '',
+                    MatchersV3.regex('text\\/plain(; charset=utf-8)?', 'text/plain')
+                  )
```

Comment:
no contenttype and length for folders? is that the same in oc10? if not we need to open a bug report

Reply:
Yeah, same for both oC10 and oCIS